content/docs/connect/connection-pooling.md

```diff
@@ -5,28 +5,51 @@ redirectFrom:
 
-With connection pooling enabled, Neon can handle up to 1000 concurrent connections. Neon uses `PgBouncer` in `transaction mode`. For limitations associated with `transaction mode`, see [Connection pooling notes and limitations](#connection-pooling-notes-and-limitations). For more information about `PgBouncer`, refer to [https://www.pgbouncer.org/](https://www.pgbouncer.org/).
+Neon uses `PgBouncer` in `transaction mode`. For limitations associated with `transaction mode`, see [Connection pooling notes and limitations](#connection-pooling-notes-and-limitations). For more information about `PgBouncer`, refer to [https://www.pgbouncer.org/](https://www.pgbouncer.org/).
 
 ## Enable connection pooling
 
-In Neon, connection pooling is configured for individual compute endpoints. It is disabled by default. You can enable connection pooling by editing an compute endpoint.
+In Neon, a database resides on a branch, and you connect to the database via the compute endpoint associated with the branch. You can enable connection pooling for all connections to a compute endpoint or for individual connections. Both methods are described below.
+
+### Enable pooling for all connections
+
+This method enables connection pooling for a compute endpoint. All connection requests to the compute endpoint are directed to a connection pooler port. Direct connections to a database through the compute endpoint are not permitted.
 
 To enable connection pooling for a compute endpoint:
 
 1. Navigate to the [Neon console](https://console.neon.tech/).
-2. On the **Dashboard**, select **Branches**.
-3. Select the branch with the compute endpoint you want to edit.
-3. Click the kebab menu in the **Endpoint** table, and select **Edit**.
-5. Toggle **Pooler enabled** to the on position.
-6. Click **Save**.
+1. On the **Dashboard**, select **Branches**.
+1. Find the branch with endpoint you want to enable pooling for, click the kebab menu in the **Compute endpoints** table, and select **Edit**.
+1. Toggle **Pooler enabled** to the on position.
+1. Click **Save**.
+
+You can also enable connection pooling when [creating a compute endpoint](/docs/manage/endpoints#create-a-compute-endpoint).
+
+### Enable pooling for individual connections
+
+This method enables pooling for individual connections that specify a `-pooler` option. Connection requests that use the `-pooler` option are directed to a connection pooler port. Connections that do not use the `-pooler` option connect directly to the database. This method supports workflows that require both pooled and non-pooled connections to the same database.
+
```

Comment:
I'd first define what `-pooler` option means. It isn't actually an option, it's a suffix for the endpoint id in the host subdomain. Not sure how to properly word it, but for now it pops out from nowhere

Also this

> option are directed to a connection pooler port

sounds like internal implementation detail. I think we'd better just state with simple words that without this suffix you get a direct connection to the Postgres, and with connection pooler is used.

Reply:
Thank you. I've revised the wording, drop mention of a connection pooler port, and changed "option" to "suffix"